editoast/src/models/fixtures.rs

```diff
@@ -214,6 +221,63 @@ pub async fn create_rolling_stock_livery(
         .expect("Failed to create rolling stock livery")
 }
 
+pub fn create_towed_rolling_stock() -> TowedRollingStock {
+    TowedRollingStock {
+        name: "TOWED_ROLLING_STOCK".to_string(),
+        label: "towed".to_string(),
+        mass: 50000.0,
+        length: 30.0, // m
+        comfort_acceleration: 0.2,
+        startup_acceleration: 0.06,
+        inertia_coefficient: 1.05,
+        rolling_resistance: RollingResistance {
+            rolling_resistance_type: "davis".to_string(),
+            A: 1.0,
+            B: 0.01,
+            C: 0.0002,
+        },
+        gamma: Gamma {
+            gamma_type: "CONST".to_string(),
+            value: 1.0,
+        },
```

Comment:
It would be a good practice to specify all the units here, but it can be done in another PR IMO